sourcemash/frontend/static/js/views/browse_view.js

```diff
@@ -0,0 +1,66 @@
+Sourcemash.Views.BrowseView = Backbone.View.extend({
+  template: JST['browse'],
+
+  initialize: function (options) {
+    var ExtendedTypeahead = Backbone.Typeahead.extend({
+      template: JST['new-feed-form'],
+    });
+
+    this.typeahead = new ExtendedTypeahead({collection: this.collection, key: 'title'});
```

Comment:
No. The form should be there, and the Browse page will eventually just have a search bar + modal “did you mean”/“do you want to add”? 